src/components/navigation/nav.module.scss

```diff
@@ -0,0 +1,365 @@
+      }
+
+      li{
+          display: inline-block;
+          padding: 10px 40px;
+          transition: all 0.4s ease 0s;
+          font-size: 14px;
+          cursor: pointer;
+          &:hover{
+            background-color: $yellow;
+          }
+      }
+  }
+  .bars{
+      display: none;
+  }
+  input{
+      padding: 8px 25px;
+      border-radius: 5px;
+      border: none;
+      outline: none;
+      background-color: #9999996f;
+  }
+  .house{
+    display: none;
+  }
+  .light{
+    cursor: pointer;
+  }
+}
+
+.m__logo {
+  display: none;
+}
+.hidden {
+  display: none;
+}
+.nav {
+  z-index: 10;
+  position: absolute;
+  top: 48px;
+  overflow-y: auto;
+  width: 248px;
+  height: calc(100% - 48px);
+  background-color: #FFF;
+  box-shadow: 0 2px 2px 0 rgba(0,0,0,0.14),0 3px 1px -2px rgba(0,0,0,0.12),0 1px 5px 0 rgba(0,0,0,0.2);
+  &__label {
+    font-weight: 300;
+    display: block;
+    text-transform: uppercase;
+    letter-spacing: .05em;
+    font-size: 12px;
+    color: #B5B5B5;
+    padding: 15px 24px;
+  }
+  &__item {
+    font-weight: 400;
+    display: block;
+    font-size: 14px;
+    color: #757575;
+    padding: 11px 25px;
+    display: flex;
+    align-items: center;
+    &.overview {
+      color: #0288D1;
+      border-bottom: 1px solid #E4E4E4;
+      background-color: #FAFAFA;
+      height: 54px;
+      margin-bottom: 12px;
+    }
+    &:hover {
+      background-color: #ECECEC;
+      cursor:pointer;
+    }
+    i {
+      display: flex;
+      justify-content: center;
+      width: 15px;
+      margin-right: 20px;
+    }
+  } 
+
+  .nav__item{
+    span{
+      padding-left: 10px;
+    }
+}
+}
+
+.all{
+  margin-top: 1.3%;
+  display: flex;
+  align-items: center;
+  gap: 25px;
+  margin-bottom: 18px;
+  .searchContainer {
+    position: relative;
+    .searchIcon {
+      position: absolute;
+      cursor: pointer;
+      top: 50%;
+      right: 10px;
+      transform: translateY(-50%);
+      font-size: 18px;
+      color: #3c2a2a;
+    }
+  }
+}
+@media only screen and (max-width: 1200px) {
+  .top__bar{
+    nav{
+    li{
+      padding: 0px 2vw;
+      }
+}
+}
+}
+@media only screen and (max-width: 1090px) {
+  .top__bar{
+    .all{
+        align-items: center;
+        gap: 15px;
+        
+    }
+      .logo__icon{
+        display: none;
+      }
+  }
+}
+@media  screen and (max-Width:850px) {
+  .top__bar{
+    nav{
+    display: none;
+}
+
+input{
+    width: 40vw;
+    border-radius: 5px;
+    border: none;
+    background-color: #9999996f; 
+   }
+.searchContainer{
+  margin-top: 1.2vh;
+}
+  .logo__icon{
+    display: none;
+  }
+
+    .house{
+      display: block;
+    }
+}
+.right_nav{
+  nav{
+    display:block
+  }
+}
+}
+@media  screen and (max-Width:600px) {
+  .top__bar{
+    strong {
+      display: none;
+  }
+  input{
+    width: 60vw;
+    border-radius: 5px;
+    border: none;
+    background-color: #9999996f; 
+   }
+  }
+}
+
+@media  screen and (max-Width:500px) {
```

Comment:
this is not true. use responsivess from variables and mixins listed there plz. check something like ```
mq($xlg){ the you can add your CSS here, that will work on devices below 1440px
}
```
check well in the mixins